include/open62541/types.h

```diff
@@ -200,6 +200,9 @@ UA_String_fromChars(const char *src) UA_FUNC_ATTR_WARN_UNUSED_RESULT;
 UA_Boolean UA_EXPORT
 UA_String_isEmpty(const UA_String *s);
 
+UA_String UA_EXPORT
```

Comment:
Add a short comment to explain that printf format is used. Also explain that memory is allocated to match the formatted string size.

make a second version that takes a preallocated buffer as the first argument.